asad_recursion_homework/recursion_assignment.js

```diff
@@ -0,0 +1,170 @@
+const assert = require('assert')
+const LinkedList = require('./linkedlist')
+
+printLinkedListInReverse=(node)=>{
+    if(node === null){
+        return
+    }
+    printLinkedListInReverse(node.next)
+    console.log(node.data)
+} 
+
+printLinkedList=(node)=>{
+    if(node === null){
+        return
+    }
+    console.log(node.data)
+    printLinkedList(node.next)
+
+}
+
+
+
+var arr = []
+print_0toN=(n)=>{
+    if(n < 0){ 
+        return//basecase}
+    }
+    print_0toN(n-1)//hypothesis
+    arr.push(n)
+    console.log(n)// induction
+}
+
+
+print_Negative_N_to_N=(n)=>{ //2
```

Comment:
What is the order of print output?
What do we need to change in this program to output in this order? if n=4 then -4, -3, -2, -1, 0, 1, 2, 3, 4